ospl.xml

```diff
@@ -0,0 +1,56 @@
+<OpenSplice>
+    <Domain>
+        <Name>ospl_sp_ddsi</Name>
+        <Id>0</Id>
+        <SingleProcess>true</SingleProcess>
+        <Description>Stand-alone 'single-process' deployment and standard DDSI networking.</Description>
+        <Service name="ddsi2">
+            <Command>ddsi2</Command>
+        </Service>
+        <Service name="durability">
+            <Command>durability</Command>
+        </Service>
+        <Service enabled="false" name="cmsoap">
+            <Command>cmsoap</Command>
+        </Service>
+    </Domain>
+    <DDSI2Service name="ddsi2">
+        <General>
+            <NetworkInterfaceAddress>${LSST_DDS_INTERFACE}</NetworkInterfaceAddress>
```

Comment:
We spoke in slack about this but, just to make sure, this should probably be 

```
<NetworkInterfaceAddress>"${LSST_DDS_INTERFACE:-AUTO}"</NetworkInterfaceAddress>
```

Reply:
Fixed